.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ruby:3.2.1
 # install dependencies
 RUN apt-get update -qq && apt-get install -y imagemagick shared-mime-info libvips && apt-get clean
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash \
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash \
```

Comment:
Could you use move node update to separate PR ?

Reply:
Ok :+1:

Reply:
updated !